triggers/README.md

```diff
@@ -17,14 +29,31 @@ We use Google Cloud Build (GCB) triggers to test the applications.
         trigger.
     *   `[PROJECT_ID]` is the GCP project ID where the trigger will be created.
 
+## Export an existing trigger
+
+*   Run the following command to export an existing trigger to file for your
+    Kubernetes application:
+
+    ```shell
+    gcloud alpha builds triggers export [TRIGGER] \
+      --destination k8s/[APP_NAME].yaml \
+      --project [PROJECT_ID]
+    ```
+
+    Where:
+
+    *   `[TRIGGER]` is an ID of the trigger or fully qualified identifier for
+        the trigger.
+    *   `[APP_NAME]` is the application name for which you want to export the
+        configuration.
+    *   `[PROJECT_ID]` is the GCP project ID where the trigger will be exported.
```

Comment:
*   `[PROJECT_ID]` is the GCP project ID of the project where the trigger will
        be exported.